components/collective-page/sections/Events.js

```diff
@@ -86,7 +86,7 @@ class SectionEvents extends React.PureComponent {
                 collective={collective}
                 event={event}
                 hideContributors={hasNoContributorForEvents}
-                disableCTA={!collective.isActive || !event.isActive}
```

Comment:
I don't get why we need to set this here, since `ContributeEvent` already do the same check?

Reply:
Ops, yes you are correct, this prop isn't used and I've removed it. 👍🏽